### PR TITLE
Task 4 to Master

### PR DESCRIPTION
**Pull request by Gaidukevich-Kirill**

### CODE REVIEW

File: Task4/ChatServer/ClientObject.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+using System.Net.Sockets;
+using System.Text;
+
+namespace ChatServer
+{
+    public class ClientObject
+    {
+        public string Id { get; set; }
+
+        public NetworkStream Stream { get; set; }
```
Comment (Shumilov-Denis) [inline Task4/ChatServer/ClientObject.cs]: Do we really need public 'set' for this property?

File: Task4/ChatServer/ClientObject.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+        public List<string> UserMessagess = new List<string>();
+
+        public readonly TcpClient Client;
+
+        public readonly ServerObject Server;
+
+        public string UserName;
```
Comment (Shumilov-Denis) [inline Task4/ChatServer/ClientObject.cs]: For public filed try use property, not fields

File: Task4/ChatServer/ClientObject.cs (hunk 1 above)
Comment (Gaidukevich-Kirill) [reply Task4/ChatServer/ClientObject.cs]: > Do we really need public 'set' for this property?

Yes.

Comment (Gaidukevich-Kirill) [reply Task4/ChatServer/ClientObject.cs]: fixed

**Pull request merged**